front/app/components/admin/PostManager/components/PostTable/AssigneeSelect.tsx

```diff
@@ -37,46 +52,42 @@ const AssigneeSelect = ({ projectId, assigneeId, onAssigneeChange }: Props) => {
         .filter((assignee) => assignee.id !== authUser.data.id)
         .map((assignee) => ({
           value: assignee.id,
-          text: formatMessage(messages.assignedTo, {
+          label: formatMessage(messages.assignedTo, {
             assigneeName: `${assignee.attributes.first_name} ${assignee.attributes.last_name}`,
           }),
-          className: 'e2e-assignee-filter-other-user',
         }));
 
       // Order of assignee filter options:
       // Assigned to me > Unassigned > Assigned to X (other admins/mods)
       return [
         {
           value: authUser.data.id,
-          text: formatMessage(messages.assignedToMe),
-          id: 'e2e-assignee-select-assigned-to-user',
+          label: formatMessage(messages.assignedToMe),
         },
         {
           value: 'unassigned',
-          text: formatMessage(messages.noOne),
-          id: 'e2e-assignee-select-unassigned',
+          label: formatMessage(messages.noOne),
         },
```

Comment:
These IDs weren't actually being used in any E2E tests, so I've just removed them. Could be nice to add tests in the future, but felt outside the scope for this ticket.